src/extra/AutoUI/models/helpers.ts

```diff
@@ -136,12 +147,87 @@ export const autoUIAddToSchema = (
 	};
 };
 
-export const getPropertyRefScheme = (
+export const getPropertyScheme = (
 	schemaValue: JSONSchema | JSONSchemaDefinition,
 ) => {
 	return typeof schemaValue !== 'boolean' &&
 		!!schemaValue.description &&
 		isJson(schemaValue.description)
-		? JSON.parse(schemaValue.description!)['x-ref-scheme']
+		? JSON.parse(schemaValue.description!)['x-foreign-key-scheme'] ??
+				JSON.parse(schemaValue.description!)['x-ref-scheme']
 		: null;
```

Comment:
Let's avoid parsing the JSON so many times:
```suggestion
	if (typeof schemaValue === 'boolean' || !schemaValue.description) {
		return null;
	}
	try {
		const json = JSON.parse(schemaValue.description!);
		return json['x-foreign-key-scheme'] ?? json['x-ref-scheme']
	} catch (err) {
		return null;
	}
```